jbpm-wb-integration/jbpm-wb-integration-client/src/main/resources/org/jbpm/workbench/wi/client/i18n/Constants.properties

```diff
@@ -94,5 +94,13 @@ InvalidPom=Corrupted file content
 UploadFailed=Upload of service task failed
 InternalUploadError=Unable to add service task due to internal error
 AddTaskSuccess=Service tasks successfully added - 
+SkippedMessage=Service tasks skipped - 
```

Comment:
Skipped tasks are tasks that were already installed?

Reply:
yes

Reply:
Perhaps then use a message to better reflect that, something like:

A service task with name {0} is already installed, skipping installation.

@mswiderski @jstastny-cz wdyt?

Reply:
+1

Reply:
There will have many service tasks, So change to "Those service task with names {0} are already installed, skipping installation.", is it ok?
@mswiderski @cristianonicolai

Reply:
_The following service tasks have been skipped as the repository already contains services with the same name as: {0}_

Reply:
done